opentelemetry/proto/trace/v1/trace.proto

```diff
@@ -329,5 +343,10 @@ enum SpanFlags {
   // Bits 0-7 are used for trace flags.
   SPAN_FLAGS_TRACE_FLAGS_MASK = 0x000000FF;
 
-  // Bits 8-31 are reserved for future use.
+  // Bits 8 and 9 are used to indicate that the parent span or link span is remote.
+  // Bit 8 indicates whether the value is known. Bit 9 indicates whether the span
+  // or link is remote.
+  SPAN_FLAGS_CONTEXT_IS_REMOTE_MASK = 0x00000300;
```

Comment:
I would split this into 2 MASKs:

```
  // Bits 8 and 9 are used to indicate that the parent span or link span is remote.
  // Bit 8 (`HAS_IS_REMOTE`) indicates whether the value is known.
  // Bit 9 (`IS_REMOTE`) indicates whether the span or link is remote.
  SPAN_FLAGS_CONTEXT_HAS_IS_REMOTE_MASK = 0x00000100;
  SPAN_FLAGS_CONTEXT_IS_REMOTE_MASK = 0x00000200;
```

Reply:
I think we should merge this PR as-is. There has been far too much discussion already, and we are now (since some time already) fussing over details that are not really important. I do think the isRemote flag in the protocol has value for backends, I believe how exactly we represent the flag is not really important.

This particular detail can also be added later on (keeping the SPAN_FLAGS_CONTEXT_IS_REMOTE_MASK  as the union of the two proposed flags and finding a different name for the proposed SPAN_FLAGS_CONTEXT_IS_REMOTE_MASK )

Reply:
Per my previous comments, since we are not allowed to change enum names and these helpers are in my opinion a source of long debate, I proposed before, and repeat I am happier to not have them initially than adding them and be inconsistent with others. So because of that I would just repeat, I am ok to merge this without the helper enums and add them later, but I am not ok to have them inconsistent with other masks.

Reply:
Assuming the latest comments are advocating for, and agree on, having the mask split into two, I've made the change. Please take a look @bogdandrutu and @Oberon00